cli/src/app.rs

```diff
@@ -184,7 +184,6 @@ pub fn app() -> Command {
             Command::new("package").about("Retrieve the details of a specific package").args(&[
                 Arg::new("name").value_name("name").help("The name of the package.").required(true),
                 Arg::new("version")
-                    .action(ArgAction::SetTrue)
```

Comment:
This is why manual repetitive tasks are always terrible ideas as a migration strategy, probably thought this was the `-v` flag.